src/components/BreakoutRoomsEditor/BreakoutRoomsEditor.vue

```diff
@@ -177,6 +177,10 @@ export default {
 	justify-content: flex-start;
 	align-items: flex-start;
 
+	h2 {
+		margin-top: 0;
+	}
```

Comment:
Are you sure it's needed ? it's 0 by default

Reply:
I see the default styles from apps.css

![image](https://github.com/nextcloud/spreed/assets/93392545/52545d5c-4e13-402b-916b-673dc5099922)
![image](https://github.com/nextcloud/spreed/assets/93392545/28ae5478-32cf-4de0-aa61-f95bbf98ffab)